projects/fluentci.io/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: https://github.com/fluentci-io/fluentci/archive/refs/tags/v{{ version }}.tar.gz
+  strip-components: 1
+
+display-name: fluentci
+
+versions:
+  github: fluentci-io/fluentci
+
+dependencies:
+  dagger.io: ^0.8.8
```

Comment:
does it have a runtime dep on dagger? but not deno?

Reply:
I just added, Deno was missing
I got this message: `pkgx: cyclic dep: deno.land: fluentci.io`, isn't that a problem?

Reply:
Well, it doesn't need the build dep with the runtime dep. I assume that's why it thinks the dep is cyclic. But a cyclic dep isn't a game stopper, just requires bootstrapping.